package-lock.json

```diff
@@ -31843,7 +31843,7 @@
         "react-dom": "^18.2.0"
       },
       "peerDependencies": {
-        "@types/react": "17 || 18"
+        "@types/react": "17 || 18 || 19"
```

Comment:
I would expect a few more changes here. Does the package lock need to be regenerated?

Reply:
The lock change for ssr-react was apparently missing, added what "npm install" updates